lighthouserc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "ci": {
+    "assert": {
+      "assertions": {
```

Comment:
How did you decide on these assertions?

Reply:
Some were in examples and some are prominent in the lighthouse report. But we can definitely tweak it to what we want
![image](https://github.com/user-attachments/assets/b3f18e65-4c91-4931-81a0-19206e835cfc)